mono/metadata/mono-security.c

```diff
@@ -65,9 +65,13 @@
  */
 static size_t mono_sysconf (int name)
 {
+#ifdef HAVE_GETEUID
```

Comment:
Incorrect ifdef. One needs to read posix & various manuals to see how portable each function is. It might be better to assume things are there when they usually are, despite posix. Otherwise future porters get to debug each of these silent changes.